modules/network-firewall/variables.tf

```diff
@@ -28,3 +28,8 @@ variable "name_prefix" {
   type        = string
 }
 
+variable allowed_public_restricted {
```

Comment:
Maybe make it clearer in the variable name / documentation that these are CIDR blocks.

Reply:
Good point. Updated with a suggestion to name it `allowed_public_restricted_subnetworks`.